### PR TITLE
[en] Not found: someone looking / anyone experiencing #2887

### PR DESCRIPTION
**Pull request by AzadehSafakish**
Fix for issue #2887.
Handles the verbs 'do' and 'make.'

Takes care of sentences like:

Does anyone experiencing any problems now?
Did I missing anything?
Could you please make someone looking into it?
I know it's an old thread, but it made me thinking.
Also, can I make it looping forever?

### CODE REVIEW

**Comment by AzadehSafakish** [comment]
TO DO: add verb 'help'

File: languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml [hunk 1]
```diff
@@ -21171,6 +21171,64 @@ USA
...
+                <example>I'll think more closely about the resource cost of my questions before making them going forward.</example>
+                <example>...and most who did make it being way behind.</example>
+            </rule>
+            <rule>
+                <pattern>
+                    <token inflected="yes">do</token>
+                    <token postag="PRP" />
+                    <marker>
+                        <token postag="VBG">
+                            <exception regexp="yes">according|being|breaking|going|having|using</exception>
```
Comment (udomai) [inline languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml:21219]: compare `&adjectives_with_ing;` (remember regexp="yes")

**Comment by AzadehSafakish** [comment]
Already in the right place, added on to entity.

File: languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml [hunk 2]
```diff
@@ -21171,6 +21171,64 @@ USA
...
+                    <token>behind</token>
+                </antipattern>
+                <pattern>
+                    <token inflected="yes">make</token>
+                    <token postag="PRP">
+                        <exception>everyone</exception>
+                    </token>
+                    <marker>
+                        <token postag="VBG">
+                            <exception regexp="yes">annoying|appealing|binding|boring|challenging|confusing|daunting|deserving|doing|embarassing|embarrassing|exciting|frustrating|interesting|pleasing|relaxing|rewarding|scaling|sickening|willing</exception>
```
Comment (udomai) [inline languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml:21201]: How about replacing this entire word collection with `&adjectives_with_ing;`?

Comment (AzadehSafakish) [reply languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml:21201]: Replaced.

**Pull request merged**